RELEASE-NOTES.md

```diff
@@ -1,5 +1,29 @@
 # Prefect Release Notes
 
+## Release 2.7.9
+
+### Enhancements
+- Add "head" flag to `flow-run logs` CLI command to limit the number of logs returned — https://github.com/PrefectHQ/prefect/pull/8003
```

Comment:
```suggestion
- Add `--head` flag to `flow-run logs` CLI command to limit the number of logs returned — https://github.com/PrefectHQ/prefect/pull/8003
```

Reply:
@peytonrunyan updated my suggestion

Reply:
@madkinsz just saw your comment. Yeah it could be 2 items

Reply:
It's essentially one item isn't it? `num_logs` doesn't really do much without `head`, right?

Reply:
It sure does, actually!
`prefect flow-run logs <flow_run_id> -n 5` is valid.
For more info:
https://github.com/PrefectHQ/prefect/pull/8003#pullrequestreview-1235208107

Reply:
It's a bit late, but we might want to update the `help` to clarify that
```
    num_logs: int = typer.Option(
        None,
        "--num-logs",
        "-n",
        help=f"Number of logs to show when using the --head flag. If None, defaults to {LOGS_WITH_LIMIT_FLAG_DEFAULT_NUM_LOGS}.",
        min=1,
    ),
    ```

Reply:
In summary, a new item for this functionality could be:
Add `num_logs` option to `flow-run logs` CLI command to specify the number of logs returned.

Reply:
Oof, I missed that. [A user is implementing the `--tail` option](https://github.com/PrefectHQ/prefect/issues/7999) so that may be a good opportunity to make the change? Unless @madkinsz do you think it's best to quickly fix it now?